styles/volunteer.scss

```diff
@@ -0,0 +1,4 @@
+#bigLogo{
```

Comment:
is the intention to change the blend-mode for only specific animations/specific pages? if not it may be better to move this to `style.scss`

this file is also probaby going to get deleted again with the new get involved page and/or cause a merge conflict

Reply:
Yes it is. The home and FAQ do not need a blend mode.